### ISSUE TITLE
v6: Form state is empty using @connect 

### ISSUE THREAD

**Issue by anaibol** [erikras/redux-form#945]
On the last realease, alpha 8, the form state seems to be empty using connect:

![image](https://cloud.githubusercontent.com/assets/3210177/15187389/3fff1790-17a1-11e6-9980-2909eeb63f80.png)

```javascript
@connect(state => {
  console.log(state);
  return ({ type: state.form.adWizard.values.type })
})
@reduxForm({ form: 'adWizard' })
export default class extends Component {
```

**Comment by erikras** [comment]
The form structure in the reducer doesn't get initialized until the first `redux-form` action. This has been the behavior for a long time.

However, with `v6`, the pattern you are using will be very common. One idea I have to help with this is to have a `getFormValue()` selector that would fail nicely to `undefined` when the value isn't there yet.

```js
import { reduxForm, getValue } from 'redux-form';

@connect(
  state => ({
    type: getValue('adWizard', 'type')
  })
)
@reduxForm({ form: 'adWizard' })
export default class AdWizardForm extends Component {
}
```

But it doesn't exist it. 😦 Would be a good first feature for someone to PR.

**Comment by browniefed** [comment]
This would essentially be a replacement for the `values` props that was passed down previously correct?
In your example not sure how `getValue` is actually accessing store state there?

**Comment by erikras** [comment]
Published my "form value selector" API as [`v6.0.0-alpha.11`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.11). Feedback welcome!
